odl/operator/oputils.py

```diff
@@ -243,6 +243,44 @@ def rmatvec(v):
                                               rmatvec=rmatvec,
                                               dtype=dtype)
 
+
+def as_cvx_operator(op, norm_bound=None):
```

Comment:
Sounds like a good idea to me :) No one (I guess...) is going to use `as_cvx_operator` without solving an optimization problem. So the solvers package is more natural in that sense.